setup.py

```diff
@@ -81,14 +81,15 @@ def run(self):
 
 setup(
     name='publicsuffix2',
-    version='2.20190205',
+    version='2.20190808',
     license='MIT and MPL-2.0',
     description='Get a public suffix for a domain name using the Public Suffix '
         'List. Forked from and using the same API as the publicsuffix package.',
+    long_description_content_type = 'text/x-rst',
     long_description='%s\n%s' % (read('README.rst'), read('CHANGELOG.rst')),
     author='nexB Inc., Tomaz Solc and David Wilson',
-    author_email='info@nexb.com',
-    url='https://github.com/nexB/python-publicsuffix2',
+    author_email='',
+    url='https://github.com/nextb/python-publicsuffix2',
```

Comment:
May be you can leave these unchanged now

Reply:
huh.. i wonder how I did that. :) fixed.